person.rb

```diff
@@ -11,6 +12,7 @@ def initialize(age, name = 'Unknown', parent_permission: true)
     @name = name
     @age = age
     @parent_permission = parent_permission
+    @rental = []
```

Comment:
Hi There 👋🏾 
as required for the person.rb file your Getter and Setter for @rentals should be present please add it and submit again 🥂